src/pages/[platform]/build-a-backend/auth/customize-auth-lifecycle/custom-auth-flows/index.mdx

```diff
@@ -66,7 +60,7 @@ For more information on adding capabilities to your application, see [Xcode Capa
 
 The custom auth flow can be [configured manually](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-lambda-challenge.html).
 
-If you have already configured custom auth, you can use the custom auth flow by changing the `authenticationFlowType` value in your [Amplify configuration](/[platform]/build-a-backend/auth/existing-resources/) to `CUSTOM_AUTH`.
+If you have already configured custom auth, you can use the custom auth flow by changing the `authenticationFlowType` value in your Amplify configuration to `CUSTOM_AUTH`.
```

Comment:
Where can I find this Amplify Configuration? Perhaps an example of setting up the configuration to CUSTOM_AUTH

Reply:
It is not moved yet. This will be post GA.

Reply:
As per my conversation with @josefaidt, customers should not touch the generated config files and he is removing these in his PR. 

https://github.com/aws-amplify/docs/pull/7444/files#diff-899d11a7738046192907b642b11d63ab2fb1f90b7e0927d5a5392af79dc3a3ab